src/style.css

```diff
@@ -0,0 +1,122 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__product-cards {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  margin: 16px 3px 0;
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  object-fit: contain;
+  object-position: center;
+}
+
+.card__name {
+  font-weight: 500;
+  color: #060b35;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 0 0;
+}
+
+.card__num {
+  color: #616070;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 4px 0 0;
+}
+
+.card__stars-line {
+  display: flex;
+  margin: 16px 0 0;
+  justify-content: space-between;
+}
+
+.stars-group {
+  display: inline-flex;
+  align-items: center;
+}
+
+.stars-group__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+}
+
+.stars-group__star:nth-child(-n+4) {
```

Comment:
create modifiers just as you did in stars task

Reply:
done